src/components/sbb-action-group/sbb-action-group.stories.js

```diff
@@ -0,0 +1,264 @@
+import { h } from 'jsx-dom';
```

Comment:
Can you maybe add the control for the size property which passes its size into the sbb-button?